elfeed-protocol-ttrss.el

```diff
@@ -340,11 +397,15 @@ it with the result entries as argument.  Return parsed entries."
                                     (autotags (elfeed-protocol-feed-autotags proto-id feed-url))
                                     (fixtags (elfeed-normalize-tags
                                               autotags elfeed-initial-tags))
+                                    (category-id (elfeed-protocol-ttrss--get-subfeed-category-id host-url feed-id))
+                                    (category-name (elfeed-protocol--ttrss--get-category-name host-url category-id))
                                     (tags (progn
                                             (unless unread
                                               (setq fixtags (delete 'unread fixtags)))
                                             (when starred
                                               (push elfeed-protocol-ttrss-star-tag fixtags))
+                                            (when category-name
```

Comment:
How about add an option named `elfeed-protocol-ttrss-fetch-category-as-tag` and use it here to allow user to disable this feature. Set the default value to `t` is just fine.

Reply:
done